### PR TITLE
Added release instructions

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -197,6 +197,18 @@ $ mvn clean install -Pdistribution,sample
...
 link:https://github.com/hawkular/hawkular-ui-components[hawkular-ui-components] in order to run the UI.
 
+== Releasing
+
+To perform a release of Hawkular Accounts, you'll need push
+permissions to the GitHub repository and execute the following:
+
+[source,bash]
+----
+mvn release:prepare release:perform -Psample,distribution
```
Comment (pilhuhn) [inline README.adoc:207]: While it is good to have those instructions, could we come to a mode where for all Hawkular modules the command is the very same to release?

**Comment by lucasponce** [comment]
Thx for adding release steps on the readme.
+1 from my side

File: README.adoc (hunk 1 above)
Comment (jpkrohling) [inline README.adoc:207]: Sure, if we can come to a common ground that would be indeed great. For accounts, we can either make the "sample,distribution" part of the set of profiles that are activated by default, or we could make use of a special release profile ("-Prelease"). As it's very simple on Accounts' side, we are quite flexible here.

Comment (jpkrohling) [inline README.adoc:207]: Created https://issues.jboss.org/browse/HAWKULAR-287 to track it.

**Pull request merged**